jobserver/api/releases.py

```diff
@@ -36,9 +37,20 @@ def initial(self, request, *args, **kwargs):
     def perform_create(self, serializer):
         data = serializer.data
 
-        message = f"{data['created_by']} released outputs from {data['path']}"
+        if "files" in data:
+            files = data["files"]
+            message = [
+                f"{data['created_by']} released {len(files)} outputs from {data['path']}:"
+            ]
+            for f in files:
+                message.append(f"`{f}`")
```

Comment:
ffffff 😂

A str.join() might be slightly less confusing here? or maybe just renaming the loop variable? Not entirely sure it would improve things though!

Reply:
me neither! I'll land as is and fix when we update this notification code